packages/kotti-ui/source/kotti-field/KtField.vue

```diff
@@ -43,7 +44,11 @@
 					<slot name="container-left" />
 				</div>
 				<slot name="container">
-					<div ref="inputContainerRef" class="kt-field__input-container">
+					<div
+						:id="`${field.inputProps.id}-input-container`"
```

Comment:
out of curiosity, what was the point of saying `input-container here?

if we want to identify on DOM level what this is, might it not be better to say 
```
id=`kt-field-${inputProps.id}`
```

Reply:
actually since it's autoGenerated and not user controlled, i'd follow the same pattern we did in the formId generation when it wasn't provided by user-app

`autoGeneratedFieldInputId.${id}`